src/Pages/login/login.jsx

```diff
@@ -0,0 +1,98 @@
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+
+export default () => {
+  const navigate = useNavigate();
+  const [userName, setUserName] = useState(null);
+  const [password, setPassword] = useState(null);
+
+  const [isUsernameActivated, setUserNameActivated] = useState(false);
+  const [isPasswordActivated, setPasswordActivated] = useState(false);
+
+  const [userNameAlert, setUserNameAlert] = useState(null);
+  const [passwordAlert, setPasswordAlert] = useState(null);
+
+  function checkUsername () {
+    if (userName == null) {
+      setUserNameAlert("Please enter user name.");
+    } else {
+      setUserNameAlert(null);
+    }
+  };
+
+  function checkPassword(password) {
+    if (password == null) {
+      setPasswordAlert("Please enter password.");
+    } else if (password.length < 5) {
+      setPasswordAlert("Password should be longer than 5 charactors.");
+    } else {
+      setPasswordAlert(null);
+    }
+  }
+
+  function handleClick() {
+    if (passwordAlert == null && userNameAlert == null) {
+      localStorage.setItem("UserName", userName);
+      navigate("/welcome");
+    }
+  }
+
+  return (
+    <div className="border flex justify-center align-center items-center bg-bgColor h-screen">
+      <div className="flex flex-col space-y-8 bg-white p-10   border-red w-[30rem] items-center rounded-lg drop-shadow ">
+        <h1 className="text-4xl font-bold text-darkRed font-sans">Log In</h1>
+        <div className="flex flex-col  space-y-4 w-full">
+          <div className="w-full">
+            <input
+              type="text"
+              placeholder="User Name"
+              value={userName}
+              onBlur={() => {
+                setUserNameActivated(true);
+
+                checkUsername(userName);
```

Comment:
This is also fine. But what i originally meant was we can have this `setUserNameActivated(true)` inside the `checkUsername` function itself as the logic is not complicated. The function name will have to change though if we do that.

No need to change now.